ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -513,6 +513,24 @@ AHU_DFSS_DSP_DX2ZC_ECONZ_EFSS_HTZC:
   - EFSS
   - HTZC
 
+AHU_SFSS_SFVSC_CHWSC_RTM_FDPM2X_ESPC_CO2DFVSC_SSPC2X_CO2M6X_ZTM_VOCM_PMM:
```

Comment:
this device makes no mention of outside air. im expecting an outside air damper on this. where is it? even if the device does not send that data, i still expect it on the model (and for the translation to be incomplete in the building config file). please address.

Reply:
Added OAFC - will remove while translation.